spec/presenters/subscription_presenter_spec.rb

```diff
@@ -35,7 +35,7 @@
       let(:search_criteria) { { working_patterns: %w[part_time] } }
 
       it "formats and returns the working pattern" do
-        expect(presenter.filtered_search_criteria["working_patterns"]).to eq("Part-time")
+        expect(presenter.filtered_search_criteria["working_patterns"]).to eq("Part time")
```

Comment:
Are we deliberately unhyphenating them? If so there are more cases of hyphenating this lying around in the translations.

Reply:
I unhyphenated them because that's how they appear in the prototype. I'll ask Jesse!


![Screenshot 2021-09-03 at 17 18 03](https://user-images.githubusercontent.com/30624173/132037078-d9d621c5-a8c8-49d5-ac79-a364310d33ff.png)